pyproject.toml

```diff
@@ -34,6 +34,7 @@ dev = [
     "pytest-cov>=4.1.0",
     "black>=23.0.0",
     "flake8>=6.0.0",
+    "debugpy>=1.6.0",
```

Comment:
```suggestion
        "debugpy>=1.6.0",
]
docs = [
    "Sphinx>=7.0.0",
]
[project.scripts]
newgame = "newgame.main:main"
[project.urls]
Homepage = "https://github.com/AaronSteen/BeaverQuest"
Repository = "https://github.com/AaronSteen/BeaverQuest"
Issues = "https://github.com/AaronSteen/BeaverQuest/issues"
```

Reply:
Well, I fucked up the formatting... but it's the URL change on lines 45-47 I was after